api/urls.py

```diff
@@ -75,13 +75,13 @@ def to_url(self, value):
     api_router.register(r"relaynumber", RelayNumberViewSet, "relay_number")
     api_router.register(r"inboundcontact", InboundContactViewSet, "inbound_contact")
     urlpatterns += [
-        path("v1/inbound_sms", inbound_sms, name="inbound_sms"),
-        path("v1/inbound_call", inbound_call, name="inbound_call"),
-        path("v1/voice_status", voice_status, name="voice_status"),
-        path("v1/sms_status", sms_status, name="sms_status"),
-        path("v1/vCard/<lookup_key>", vCard, name="vCard"),
+        path("v1/inbound_sms/", inbound_sms, name="inbound_sms"),
+        path("v1/inbound_call/", inbound_call, name="inbound_call"),
+        path("v1/voice_status/", voice_status, name="voice_status"),
+        path("v1/sms_status/", sms_status, name="sms_status"),
+        path("v1/vCard/<lookup_key>/", vCard, name="vCard"),
```

Comment:
For POST URLs that we need to switch remotely as well, we could allow both URLs for a time:

```suggestion
        path("v1/inbound_sms", inbound_sms, name="legacy_inbound_sms"),
        path("v1/inbound_call", inbound_call, name="legacy_inbound_call"),
        path("v1/voice_status", voice_status, name="legacy_voice_status"),
        path("v1/sms_status", sms_status, name="legacy_sms_status"),
        path("v1/inbound_sms/", inbound_sms, name="inbound_sms"),
        path("v1/inbound_call/", inbound_call, name="inbound_call"),
        path("v1/voice_status/", voice_status, name="voice_status"),
        path("v1/sms_status/", sms_status, name="sms_status"),
        path("v1/vCard/<lookup_key>/", vCard, name="vCard"),
```

Reply:
Hmm, I like the idea, implemented in [b9da2cf](https://github.com/mozilla/fx-private-relay/pull/3013/commits/b9da2cfc1d821bf33489b92b1a1dc2376da0648f).

Unfortunately, my texts still don't seem to go through on the dev server, and calls receive a robot voice saying something like "An application error has occurred".

Since this was just intended to be a quick fix to solve a minor annoyance, I might close this PR, since it's starting to get into not-worth-the-hassle territory.

Although I just tried it out with `main`, and got the same error, so maybe something else is wrong at the moment.

Reply:
Never mind, there was a general error on dev, and calling and text did actually work with these changes!